distributed/deploy/spec.py

```diff
@@ -211,6 +211,9 @@ def __init__(
 
         if silence_logs:
             self._old_logging_level = silence_logging(level=silence_logs)
+            self._old_bokeh_logging_level = silence_logging(
+                level=logging.CRITICAL, root='bokeh'
```

Comment:
The `silence_logs` value here can be a logging level from the user.  We should probably respect what they're asking for.

```suggestion
                level=silence_logs, root='bokeh'
```

Reply:
Right, I did consider that. The problem is that this probably depends on the package, e.g. it seems the default value is INFO, which would be pretty verbose for bokeh and certainly a big difference to CRITICAL.

Reply:
So for my purposes I'd actually be pretty happy with that but it doesn't really jive with your desire to have interactive Python usage be pretty much silent. Maybe I'm missing something though and the default level is actually higher than INFO.

Reply:
The default level for LocalCluster (the 99% usecase) is WARN

https://github.com/dask/distributed/blob/4953f81d7b6856f6a22f00e27476d4a554a120c2/distributed/deploy/local.py#L96

I can live with what's here though.  I'm happy to defer the deicsion on what's best here to you.

Reply:
Okay, I think making it configurable is definitely better than hardcoding it so I'll apply your suggestion.